### ISSUE TITLE
Fails to download one of the dependency files

### ISSUE THREAD

**Issue by JorchRL** [timreichen/Bundler#56]
When I try to use the Bundler or cache it as a dependency, I get this error

```
Download https://esm.sh/v86/postcss-nesting@7.0.1/deno/postcss-nesting.js
error: Import 'https://esm.sh/v86/postcss-nesting@7.0.1/deno/postcss-nesting.js' failed: 500 Internal Server Error
    at https://esm.sh/v86/postcss-preset-env@6.7.0/deno/postcss-preset-env.js:2:2086
```

It seems that the file in question is not available from esm.sh. When I try to get it from the browser i get this:

```
/* esm.sh - error */
throw new Error("[esm.sh] " + "open /tmp/esm-build-2112f0f36dbc8a59c1b5aa42d9c9524d7448e54a-462f9c4f/node_modules/postcss-nesting/index.es.js: no such file or directory");
export default null;
```
So I guess it is no longer available. Which is unfortunate, because I can't seem to get Bundler to work.

**Comment by timreichen** [comment]
Deno as well as esm.sh has changed a lot over time and postcss-preset-env failed at some point. This is fixed with the restructure and will be solved with the next release.

**Comment by milotheirself** [comment]
I'm getting the same Error. When is this next release planned to be, @timreichen?

**Comment by timreichen** [comment]
Just released 0.9.0 now.
